src/BuildScriptGenerator/DotNetCore/DotnetCorePlatform.cs

```diff
@@ -139,7 +139,8 @@ public PlatformDetectorResult Detect(RepositoryContext context)
             {
                 installBlazorWebAssemblyAOTWorkloadCommand = DotNetCoreConstants.InstallBlazorWebAssemblyAOTWorkloadCommand;
                 manifestFileProperties[ManifestFilePropertyKeys.Frameworks] = "blazor";
-                this.logger.LogInformation("Detected the the following framework(s): blazor");
+                this.logger.LogInformation("Detected the following framework(s): blazor");
```

Comment:
nit: maybe create a `detectionMessage` variable that stores the string message you have for each change and is reused by the subsequent `LogInformation` and `WriteLine` calls?